Extension/package.json

```diff
@@ -506,6 +506,15 @@
           "default": "Disabled",
           "description": "If enabled, code is colorized based on IntelliSense. This setting has no effect if IntelliSense is disabled.",
           "scope": "resource"
+        },
+        "C_Cpp.dependencyManager": {
```

Comment:
Is "dependencyManager" the correct terminology? I thought it was packageManager.

Reply:
The preferred terminology has changed. Vcpkg is still recognized as a package manager, but **officially** it is a "library manager" or a bit more generally than that a "dependency manager"